### DISCUSSION TITLE
Visual discussions regarding to osu!mania "Argon" skin.

### DISCUSSION THREAD

**Discussion by IkaWaAyuMu** [ppy/osu#20616] [Q&A] [answered]
I know the skin is not final yet, so I think drop it here is ok.

- The ~~legal cheat~~ Timing-based note coloring doesn't have distinct look on each timing signature.
  - Triangle 
![image](https://user-images.githubusercontent.com/7066150/194704585-0d43c9c1-5076-46b2-a9dc-6dd14ccbc6fb.png)
  - Argon
![image](https://user-images.githubusercontent.com/7066150/194704718-77a4a778-c5bf-4917-b916-f9b2cfba2cda.png)

- When combine bright background with visual impairment mods It's insanely impair my visual a lot more than Triangle *I think hold note is unreadable at this state.* (This is only my opinion) - Image test with white background with 30% dim

  - Triangle
  ![image](https://user-images.githubusercontent.com/7066150/194705221-70e34a91-be59-46be-a22b-5882c46ab280.png)
  ![image](https://user-images.githubusercontent.com/7066150/194705340-99881189-e202-4974-b28d-d3231206793a.png)
  - Argon
  ![image](https://user-images.githubusercontent.com/7066150/194705246-d0b8ecbb-e950-4514-b57c-d9e2a89e7e7b.png)
  ![image](https://user-images.githubusercontent.com/7066150/194705357-3c02a89f-21ce-4e9a-a11f-b928bec07748.png)

EDIT : With FL fix, I finally tested with FL applied(which use other method of hiding notes), It's still well readable.
![image](https://user-images.githubusercontent.com/7066150/194755794-ff99e80f-4fb8-49bd-9c4a-f256400338df.png)

- I think hold end white line need to be align to where to release the hold, not where the block sprite end.
  - Argon 
![image](https://user-images.githubusercontent.com/7066150/194706042-e84fe91b-ac5f-496a-bca9-15c27908f51f.png)

**Comment by smoogipoo** [comment, marked as answer]
Thanks for the reports.

- https://github.com/ppy/osu/issues/20977
- https://github.com/ppy/osu/issues/20978
- https://github.com/ppy/osu/issues/20979
